hash/spectrum_tool.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0
+-->
+<softwarelist name="spectrum_tool" description="Sinclair ZX Spectrum - Tools and Utilites">
+
+	<software name="floatspy">
+		<description>Floating Bus Test v0.33</description>
+		<year>2002</year>
+		<publisher>Ramsoft</publisher>
+		<part name="floatspy" interface="spectrum_cass">
+			<dataarea name="cass" size="5782">
+				<rom name="floatspy.tap" size="5782" crc="2a4e5f1e" sha1="9b61d5055e3021a130f6ee960b9dabdb8accd13a"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="fpga48all">
+		<description>FPGA 48k</description>
+		<year>20??</year>
+		<publisher>&lt;unknown&gt;</publisher>
+		<part name="fpga48all" interface="spectrum_cass">
+			<dataarea name="cass" size="15245">
+				<rom name="FPGA48all.tap" size="15245" crc="06d0f1c7" sha1="2dc35ef01a841e139cfc93a142d2a0c3bf851d0f"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="fusetest">
+		<description>Fuse test</description>
+		<year>20??</year>
+		<publisher>&lt;unknown&gt;</publisher>
+		<part name="fusetest" interface="spectrum_cass">
+			<dataarea name="cass" size="2411">
+				<rom name="fusetest.tap" size="2411" crc="a7d5177a" sha1="fe0e27a6843328e4cf1212f0ad511d97eca5db4f"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="inttest">
+		<description>Test INT v1.10</description>
+		<year>1995</year>
+		<publisher>Good Boy</publisher>
+		<part name="inttest" interface="spectrum_cass">
+			<dataarea name="cass" size="9521">
+				<rom name="IntTest.tap" size="9521" crc="b47c794c" sha1="38eca63ef4f3ae3fc170ac2523aee4c659ed176b"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="scroll17">
+		<description>Scroll 17</description>
+		<year>2017</year>
+		<publisher>&lt;unknown&gt;</publisher>
+		<part name="scroll17" interface="spectrum_cass">
+			<dataarea name="cass" size="9168">
+				<rom name="scroll17.tap" size="9168" crc="65e4c06d" sha1="44089e9c03f357871e5293f83203771d04fd7b73"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="tactstest">
+		<description>Tact Meter v1.0</description>
+		<year>2006</year>
+		<publisher>Strunov / Good Boy</publisher>
+		<part name="tactstest" interface="spectrum_cass">
+			<dataarea name="cass" size="2622">
+				<rom name="TactsTest.tap" size="2622" crc="1012357e" sha1="51c852bacccfbfc4699012e55edb957d24f9d9c6"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="timingtest">
+		<description>Timing Tests v1.0</description>
+		<year>20??</year>
+		<publisher>&lt;unknown&gt;</publisher>
```

Comment:
`&lt;homebrew&gt;` is a better tag, namely in searching.
Also I believe all these entries are freeware software right? In that case, all the publisher nodes needs to be converted to `&lt;homebrew&gt;`, while author info can be changed with `<info name="developer" value="John Doe"/>`

Reply:
What about multiple (3) authors? Duplicate `<info name="deveper"` ?

Reply:
hash DB either uses commas or slashes
`<info name="developer" value="Park Place Productions, Season Ticket Productions" />`
`<info name="developer" value="Interactive Binary Illusions / Sub Zer0 Software" />`
I don't mind either way, as long it's consistent across the file.

Reply:
figured out

Reply:
Oops. I checked and multiple tags processed correctly. Do you want me to change it to suggested separators in one tag?

Reply:
From a readability point of view I'd stick in one tag yes. 
I'm also unsure that it will read properly from external XML parser this way, probably not?

Reply:
It will, and just checked multiple tags works better.
![image](https://user-images.githubusercontent.com/58155/194967556-673d422d-c29a-4418-8b12-6e0f6d3be228.png)
![image](https://user-images.githubusercontent.com/58155/194967504-45a359c6-8017-4550-b40e-edcd9fe57607.png)

Reply:
Arguably I don't think it looks that well, it's inconsistent with how that UI is supposed to display.
My gripe anyway was more about the readability from an human perspective with the raw XML itself, and the possible case where some XML engines will take the easy path and just output the first or the last node.

Reply:
Alright. Single entry here it is.

Reply:
No, use multiple `info` elements.  That’s how existing software lists work.  Using commas creates ambiguity.

Reply:
> No, use multiple info elements. That’s how existing software lists work. Using commas creates ambiguity.

That's news to me, I've searched the hash DB and it has commas/slashes/even some &amp; all scattered through. I also don't think I've ever seen a single entry that maps multiple `info `nodes like at all?